src/main/kotlin/no/nav/familie/ef/sak/ekstern/EksternBehandlingService.kt

```diff
@@ -31,4 +37,21 @@ class EksternBehandlingService(
             .map { it.id }
             .toSet()
     }
+
+    fun kanOppretteFørstegangsbehandling(ident: String, type: StønadType): Boolean {
+        val allePersonIdenter = personService.hentPersonIdenter(ident).identer.map { it.ident }.toSet()
+        val fagsak = fagsakService.finnFagsak(allePersonIdenter, type)
+
+        val harIngenReelleBehandlinger = fagsak?.let { harIngenReelleBehandlinger(it) } ?: true
+        val harIngenInnslagIInfotrygd = !infotrygdService.eksisterer(ident, setOf(type))
+
+        return harIngenReelleBehandlinger && harIngenInnslagIInfotrygd
+    }
+
+    fun harIngenReelleBehandlinger(fagsak: Fagsak): Boolean {
+        val reelleBehandlinger = behandlingService.hentBehandlinger(fagsak.id)
+            .filter { it.resultat != BehandlingResultat.HENLAGT }
+
+        return reelleBehandlinger.isEmpty()
```

Comment:
Her er det mulig å bruke `none` i stedet for `filter + isEmpty`, begge måter virker fint